src/features/commands.ts

```diff
@@ -303,7 +303,8 @@ How To Ask Questions The Smart Way https://git.io/JKscV
             description: `\\\`\\\`\\\`js
 // short code snippets go here
 \\\`\\\`\\\`
-
+Like Shown Below 👇
+https://im5.ezgif.com/tmp/ezgif-5-26189abd64.gif
```

Comment:
This is 404ing for me, and I'm not sure how comfortable I am linking out to a random domain like this. Is there a way to get it as en embed on the message?